opencti-platform/opencti-graphql/src/database/cache.ts

```diff
@@ -57,6 +58,7 @@ export const resetCacheForEntity = (entityType: string) => {
   const types = [entityType, ...(STORE_ENTITIES_LINKS[entityType] ?? [])];
   types.forEach((type) => {
     if (cache[type]) {
+      logApp.debug('Reset cache for entity', { type, entityType });
```

Comment:
```suggestion
      logApp.debug('Reset cache for entity', { type });
```
`types` array already contains `entity_type` as first value.

Reply:
in the log we will have "type": the entity type of the cached data, and "entityType" which is the entity type of the instance received in the event. They can be different because for example when we update a group, we will invalidate the cache of users (so entityType will be "Group" and type will be "User"). 
What do you suggest we should log ?

Reply:
Understood ! I did not catch this was in the `if (cache[type]) {` clause, so yeah you're right !